stores/viewer.js

```diff
@@ -6,17 +6,17 @@ export const use_viewer_store = defineStore('viewer', {
     picked_point: { x: null, y: null }
   }),
   actions: {
-    toggle_picking_mode(value) {
+    toggle_picking_mode (value) {
```

Comment:
You have to work on this format. Maybe add some automatic formating like prettier to force a style?

Reply:
It's the automating formatting that did that already